ui-tests/src/test/java/io/syndesis/qe/pages/integrations/edit/ActionConfigureComponent.java

```diff
@@ -8,13 +8,17 @@
 import com.codeborne.selenide.SelenideElement;
 
 import io.syndesis.qe.pages.SyndesisPageObject;
+import lombok.extern.slf4j.Slf4j;
 
-public class ActionConfigureComponent extends SyndesisPageObject {
+@Slf4j
+public abstract class ActionConfigureComponent extends SyndesisPageObject {
 
 	private static final class Element {
 		public static final By ROOT = By.cssSelector("syndesis-integrations-action-configure");
 	}
 
+	public abstract void fillInput(String value);
```

Comment:
I'm not sure if this is a good solution.

Reply:
@lvydra this is a result of comments discussion of  .../src/test/java/io/syndesis/qe/pages/integrations/edit/ActionConfigureComponentInvokeSql.java in previous pull request #221 (the same topic). 
I think - if you have some specific idea how to do it - it's always best to give a proposal.

Reply:
@Stefan365 I know I read the discussion, but I don't think that you get Davids proposal right. He has been talking about solving problem with code duplicities on sql related ActionConfigureComponent. 
And there is nice spot for general ancestor.

Reply:
@lvydra that was my above note about - if you have specific Idea, please come out with it and you prevent misunderstanding.

Reply:
@Stefan365 Well, I did it in last comment.

Reply:
done